### PR TITLE
TG-2368 Checkout customer information

### PR DESCRIPTION
**Pull request by paulkennethkent**
Checkout now CTA on cart page links to checkout page
Checkout page structure added
Validate and submit form
Redirect to payment page

### CODE REVIEW

**Review by JohnLBergqvist — CHANGES_REQUESTED** [review]
Update the copyright date to 2018 on the relevant pages.

**Review by Charliemowood — APPROVED** [review]

**Comment by paulkennethkent** [comment]
@JohnBergqvist and @Charliemowood copyright has now been updated.

**Review by JohnLBergqvist — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -16,4 +16,4 @@ To run type:
 In addition to Diffblue's normal coding standards this repository applies (via Checkstyle) Google's Java style.
 
 
-Copyright 2016-2017 DiffBlue limited. All rights reserved.
+copyright 2016-2018 DiffBlue limited. All rights reserved.
```
Comment (jgwilson42) [inline README.md]: Please could you correct the capitalisation of 'Copyright'?

Comment (jgwilson42) [reply README.md]: Please do this throughout

File: src/main/java/com/diffblue/demo/ecommerce/controllers/CheckoutController.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
+package com.diffblue.demo.ecommerce.controllers;
```
Comment (jgwilson42) [inline src/main/java/com/diffblue/demo/ecommerce/controllers/CheckoutController.java:1]: Please add a copyright notice

File: src/main/java/com/diffblue/demo/ecommerce/forms/AddressForm.java [hunk 3]
```diff
@@ -0,0 +1,62 @@
+package com.diffblue.demo.ecommerce.forms;
```
Comment (jgwilson42) [inline src/main/java/com/diffblue/demo/ecommerce/forms/AddressForm.java:1]: Please add a copyright notice

File: src/main/java/com/diffblue/demo/ecommerce/forms/CustomerForm.java [hunk 4]
```diff
@@ -0,0 +1,66 @@
+package com.diffblue.demo.ecommerce.forms;
```
Comment (jgwilson42) [inline src/main/java/com/diffblue/demo/ecommerce/forms/CustomerForm.java:1]: Please add a copyright notice

File: src/main/java/com/diffblue/demo/ecommerce/models/Cart.java [hunk 5]
```diff
@@ -1,21 +1,22 @@
-package com.diffblue.demo.ecommerce;
```
Comment (jgwilson42) [inline src/main/java/com/diffblue/demo/ecommerce/models/Cart.java:1]: Please add a copyright notice

**Review by jgwilson42 — CHANGES_REQUESTED** [review]

**Comment by paulkennethkent** [comment]
@jgwilson42 - Changes made throughout, please re-review.

**Review by jgwilson42 — APPROVED** [review]

**Pull request merged**